platform/vs/docker-sonic-vs/default_chassis_cfg.json

```diff
@@ -0,0 +1,7 @@
+{
+    "DEVICE_METADATA": {
+        "localhost": {
+            "chassis_db_address" : "10.8.1.200"
```

Comment:
I remember the HLD mentioned there was a `connect_chassis_db` field as well, do we need that here?

Reply:
No, connect_chassis_db indicates chassis_db global service is available in a virtual chassis environment, the default_chassis_cfg.json is used when the required virtual-chassis configuration is missing and sonic-vs runs as fixed switch, so that sonic-vs container won't fail due to the error of file not found.